File: guides/java/kora-java-guide-cache-multi-level-app/src/main/java/io/koraframework/guide/cache/service/UserCaffeineCache.java
package io.koraframework.guide.cache.service;

import io.koraframework.cache.annotation.Cache;
import io.koraframework.cache.caffeine.CaffeineCache;
import io.koraframework.guide.cache.dto.UserResponse;

@Cache("cache.caffeine.users")
public interface UserCaffeineCache extends CaffeineCache<String, UserResponse> {}

File: guides/kotlin/kora-kotlin-guide-cache-multi-level-app/src/main/kotlin/io/koraframework/guide/cache/service/UserCaffeineCache.kt
package io.koraframework.guide.cache.service

import io.koraframework.cache.annotation.Cache
import io.koraframework.cache.caffeine.CaffeineCache
import io.koraframework.guide.cache.dto.UserResponse

@Cache("cache.caffeine.users")
interface UserCaffeineCache : CaffeineCache<String, UserResponse>
